GVRf/Extensions/gvrf-particlesystem/build.gradle

```diff
@@ -0,0 +1,90 @@
+apply plugin: 'com.android.library'
+
+repositories {
+    flatDir {
+        dirs '../../gearvrf-libs/'
+    }
+}
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion '25.0.2'
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 21
+
+        ndk {
+            moduleName "gvrf-particlesystem"
+            abiFilter "armeabi-v7a"
+        }
+        externalNativeBuild {
+            cmake {
+                //difference in sign extension when reinterpret_cast-ing a pointer between
+                //gcc and clang hence overriding the default toolchain to match the one used
+                //by the other modules
+                arguments.add('-DANDROID_TOOLCHAIN=gcc')
+                if (rootProject.hasProperty("buildGvrfAndDemos")) {
+                    arguments.add('-DCMAKE_BUILD_TYPE=Debug')
+                }
+            }
+        }
+    }
+
+    buildTypes {
+        debug {
+            debuggable = true
+            jniDebuggable = true
+        }
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-project.txt'
+        }
+        debug {
+            jniDebuggable true
+        }
+    }
+
+    sourceSets.main {
+        java.srcDirs = ['src/main/java']
+        //jni.srcDirs = [] // no auto generation of Android.mk
```

Comment:
Mysterious commented-out code in gradle file. If we don't clean it up now, it will stay here forever and fossilize.